pkg/adapter/nimbus-netpol/main.go

```diff
@@ -40,6 +41,6 @@ func main() {
 		logger.Info("All workers finished, shutting down")
 	}()
 
-	logger.Info("Network Policy adapter started")
-	manager.ManageNetPols(ctx, nimbusPolicyCh, nimbusPolicyToDeleteCh, clusterNpChan, clusterNpToDeleteChan)
+	logger.Info("KubeArmor adapter started")
+	manager.ManageKsps(ctx, nimbusPolicyCh, nimbusPolicyToDeleteCh, nimbusPolicyUpdateCh, clusterNpChan, clusterNpToDeleteChan)
```

Comment:
This manager is responsible for managing network policies not for kubearmor policies, please revert it back.